From b21ce6002586a8e883d905601d88bfb4a3b2e789 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Tue, 28 Nov 2017 15:58:28 +0800
Subject: [PATCH 2923/5242] MLK-17012 ARM: dts: imx: update vdd_soc setpoint
 voltage on imx6sll

commit  5ad17e14c15237be792f3e57458602cfeb1b639a from
https://source.codeaurora.org/external/imx/linux-imx.git

According to the latest datasheet(Rev. 0.2, 11/2017), the
VDD_SOC_IN voltage can be set to 1.15V always, no constrain
between VDD_SOC_IN and VDD_ARM_IN, so change the voltage
of VDD_SOC_IN for 996MHz setpoint to 1.175V as other setpoints.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sll.dtsi b/arch/arm/boot/dts/imx6sll.dtsi
index c722329..0e7592b 100644
--- a/arch/arm/boot/dts/imx6sll.dtsi
+++ b/arch/arm/boot/dts/imx6sll.dtsi
@@ -58,7 +58,7 @@
 			>;
 			fsl,soc-operating-points = <
 				/* ARM kHz      SOC-PU uV */
-				996000          1275000
+				996000          1175000
 				792000          1175000
 				396000          1175000
 				198000		1175000
-- 
1.7.9.5

